readme.txt

```diff
@@ -102,19 +102,23 @@ Learn more (and get detailed instructions) in our [contribute guide](http://jetp
 
 = 3.6 =
 Release Date: July 6th, 2015
+Release Post: http://jetpack.me/2015/07/06/jetpack-3-6-updated-social-widgets-jetpack-cli-and-more/
```

Comment:
@georgestephanis looks like prior major releases already have the post linked.  Did you mean changing them to highlight featured changes first?